_posts/2019-09-05-cover-song-similarity.md

```diff
@@ -0,0 +1,321 @@
+---
+layout: post
+title: Cover song similarity algorithms in Essentia
+category: news
+---
+
+
+`Cover Song Identification` or `Music Version Identification` is a task of identifying when two musical recordings are derived from the same music composition. A cover version of a song can be drastically different from the original recording. It may have variations in key, tempo, structure, melody, harmony, timbre, language and lyrics compared to its original version, which makes it a challenging task [1]. It is typically set as a query-retrieval task where the system retrieves a ranked list of possible covers of a given query song from a reference database. We recommend you reading [1] for more comprehensive details of this task. 
```

Comment:
"which makes cover identification a challenging task"

Reply:
"possible covers for a given query song"

Reply:
"We recommend you reading [1] for more comprehensive details of this task." may be not need to mention this. Furthermore, we already cite this reference in a sentence before.